data-products/src/new_tab_recommendations/aggregate_engagement_flow.py

```diff
@@ -29,27 +32,47 @@
             CAST(flattened_tiles.id AS INT64) AS TILE_ID,
             FORMAT_DATETIME("%Y-%m-%dT%H:%M:%SZ", MAX(submission_timestamp)) as UPDATED_AT,  -- Feature Store requires ISO 8601 time format
             COUNT(*) AS TRAILING_1_DAY_IMPRESSIONS,
-            SUM(CASE WHEN click IS NOT NULL THEN 1 ELSE 0 END) AS TRAILING_1_DAY_OPENS,
-            -- For now, we only need 1 day trailing data, so leave the other ones at 0.  
-            0 AS TRAILING_7_DAY_IMPRESSIONS,
-            0 AS TRAILING_7_DAY_OPENS,
-            0 AS TRAILING_14_DAY_IMPRESSIONS,
-            0 AS TRAILING_14_DAY_OPENS,
-            0 AS TRAILING_21_DAY_IMPRESSIONS,
-            0 AS TRAILING_21_DAY_OPENS,
-            0 AS TRAILING_28_DAY_IMPRESSIONS,
-            0 AS TRAILING_28_DAY_OPENS
+            SUM(CASE WHEN click IS NOT NULL THEN 1 ELSE 0 END) AS TRAILING_1_DAY_OPENS
         FROM impressions_data
         CROSS JOIN UNNEST(impressions_data.tiles) AS flattened_tiles
         GROUP BY TILE_ID
         ORDER BY TRAILING_1_DAY_IMPRESSIONS DESC
         LIMIT 16384 -- Limit to Snowflake's max list size. There are a lot of old items that still get some impressions.
     """
 
+EXPORT_GLEAN_TELEMETRY_SQL = """
+    WITH events AS (
+      SELECT
+        submission_timestamp,
+        event.name AS event_name,
+        extra.value AS recommendation_id
```

Comment:
Isn't `extra` a json/map with potentially multiple keys/values? Curious about what looks like a single key/value pair here.

Reply:
I simplified the query by pulling `AND extra.key = 'recommendation_id'` up to the `CROSS JOIN`:
```sql
CROSS JOIN UNNEST(event.extra) AS extra ON extra.key = 'recommendation_id'
```
This produces the same result, and hopefully it's clearer that we're querying only a single extra key.